### ISSUE TITLE
Motor hour meter

### ISSUE THREAD

**Issue by petebachant** [petebachant/TurbineDAQ#71]
Log motor runtime accumulation to standalone file

**Comment by petebachant** [comment]
From @meganandd 

This might get logged in the drive

**Comment by meganandd** [comment]
Yes, confirmed that this is information stored in the drive (can access it using Yaskawa's SigmaWin+ software). I think it would be nice if it was easily accessible in the DAQ and might even be stored in drive's process data.
